docs/docs_toolkit.md

```diff
@@ -1 +1,179 @@
-# Documentation tookit
\ No newline at end of file
+# Documenting your project
+
+This section helps you add documentation to your Ansible project or improve your existing documentation.
+The onboarding [project-template](https://github.com/ansible-community/project-template)
+in GitHub gives you the scaffolding files to start your documentation effort. 
+
+## Creating new documentation projects
```

Comment:
```suggestion
## Creating new documentation sets
```

Reply:
I think we should say either "documentation set" or maybe "docsite" here. For me "documentation project" sounds like a standalone thing dedicated only to docs.